[regression] simple programs can't be compiled on cpp backend using -d:release

hello.nim:
```nim
echo "hello"
```
can't be compiled with:
```
nim cpp -d:release hello.nim
```
An error message like this one is produced :
```
/tmp/nimcache/stdlib_system.o: In function `Exception::~Exception()':
stdlib_system.cpp:(.text._ZN9ExceptionD2Ev[_ZN9ExceptionD5Ev]+0x19): undefined reference to `popCurrentExceptionEx(unsigned long long)'
/tmp/nimcache/stdlib_system.o: In function `tyObject_SystemError_Fjf9b9b2IlzRO2tspHxG3tgg::~tyObject_SystemError_Fjf9b9b2IlzRO2tspHxG3tgg()':
stdlib_system.cpp:(.text._ZN45tyObject_SystemError_Fjf9b9b2IlzRO2tspHxG3tggD2Ev[_ZN45tyObject_SystemError_Fjf9b9b2IlzRO2tspHxG3tggD5Ev]+0x19): undefined reference to `popCurrentExceptionEx(unsigned long long)'
/tmp/nimcache/stdlib_system.o: In function `tyObject_IOError_9czoZw3pzPyrBmb5zTHY1OQ::~tyObject_IOError_9czoZw3pzPyrBmb5zTHY1OQ()':
stdlib_system.cpp:(.text._ZN40tyObject_IOError_9czoZw3pzPyrBmb5zTHY1OQD2Ev[_ZN40tyObject_IOError_9czoZw3pzPyrBmb5zTHY1OQD5Ev]+0x19): undefined reference to `popCurrentExceptionEx(unsigned long long)'
collect2: error: ld returned 1 exit status
Error: execution of an external program failed: 'g++   -o hello  /tmp/nimcache/hello.o /tmp/nimcache/stdlib_system.o    -ldl'
```

This was introduced by:
```
commit 16c1a90857992df0960f53f5ffa578f14059cee4
Author: cooldome <cdome@bk.ru>
Date:   Tue Apr 10 11:14:59 2018 +0100

    Cpp codegen: handling of imported exceptions. Fixes #3571 (#7360)
```
Programs without any ```try/except``` (either explicit in the user code or produced by Nim itself) can't be compiled anymore using the cpp backend.

--- a/compiler/ccgstmts.nim
+++ b/compiler/ccgstmts.nim
@@ -572,6 +572,8 @@ proc genBreakStmt(p: BProc, t: PNode) =
   lineF(p, cpsStmts, "goto $1;$n", [label])
 
 proc genRaiseStmt(p: BProc, t: PNode) =
+  if p.module.compileToCpp:
+    discard cgsym(p.module, "popCurrentExceptionEx")
   if p.nestedTryStmts.len > 0 and p.nestedTryStmts[^1].inExcept:
     # if the current try stmt have a finally block,
     # we must execute it before reraising

